src/views/Apptour/Tourmap.vue

```diff
@@ -224,24 +240,28 @@ main {
 		position: fixed;
 		right: 0;
 		margin: 0 1em 0em 0;
-		bottom: -4em;
+		// bottom: -4em;
+		// bottom: 1em;
+		bottom: 1em;
```

Comment:
EVEN YOU DON'T USE REM ANYMORE 🤔

Reply:
Readability. No all caps because no one will want to read it. And I have changed it now👍